dev/tasks/dev.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const initTask = require( './utils/dev-init' );
+const pluginCreateTask = require( './utils/dev-plugin-create' );
+const pluginInstallTask = require( './utils/dev-plugin-install' );
+const pluginUpdateTask = require( './utils/dev-update' );
+const pluginStatusTask = require( './utils/dev-status' );
+const boilerplateUpdateTask = require( './utils/dev-boilerplate-update' );
+const ckeditor5Path = process.cwd();
+
+module.exports = ( grunt ) => {
+	const packageJSON = grunt.config.data.pkg;
+
+	grunt.registerTask( 'dev-init', function() {
```

Comment:
I'm using grunt `async()` and `options()` methods that are available in function's scope.